src/resolvers/UserResolver.ts

```diff
@@ -1,10 +1,18 @@
-import { Resolver, Mutation, Arg } from 'type-graphql';
+import { valueFromASTUntyped } from 'graphql';
+import { Resolver, Mutation, Arg, Query } from 'type-graphql';
 import { User } from '../entities/User';
 import UserRepo from '../repos/UserRepo';
+import WeeklyDebriefRepo from '../repos/WeeklyDebriefRepo';
 
 @Resolver((_of) => User)
 class UserResolver {
-  @Mutation((_returns) => User)
+  @Query((_returns) => User, { nullable: true, description: 'Returns a single [User] via a given [uuid]'})
+  async getUser(@Arg('uuid') uuid: string) {
+    return UserRepo.getUserByUUID(uuid);
+  }
+
+  @Mutation((_returns) => User, {description: `Creates a single <User> via given <deviceToken>, <followedPulbications>,
+   and <deviceType>. Will not create a user if one with the same <deviceToken> already exists.`})
```

Comment:
Should be more explicit then 'will not create a user' (then what?). Creates a single user given unique fields, otherwise does nothing.